src/fields/fft_poisson_solver/fft/WrapRocFFT.cpp

```diff
@@ -50,7 +50,6 @@ namespace AnyFFT
         fft_plan.m_real_array = real_array;
         fft_plan.m_complex_array = complex_array;
         fft_plan.m_dir = dir;
-        fft_plan.m_dim = dim;
```

Comment:
This is not used anywhere. Is it expected that we'll need it in the future?

Reply:
This is from the WrapRocFFT.cpp I got from WarpX, maybe it is needed there and not here. I think, that would be for you to decide in this case.

Reply:
Just checked, that's not relevant for Hipace++, we can remove it.